packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanel.tsx

```diff
@@ -664,23 +663,12 @@ export function TransferPanel() {
         childChainId: childChain.id
       }
 
+      addPendingTransaction(newTransfer)
       openTransactionHistoryPanel()
       setTransferring(false)
       clearAmountInput()
-
-      const depositTxReceipt = await depositForBurnTx.wait()
-      const { messageBytes, attestationHash } =
-        getAttestationHashAndMessageFromReceipt(depositTxReceipt)
-
-      if (depositTxReceipt.status === 0) {
```

Comment:
We don't need this. Tx updater checks for it, and if it's 0 it will show tx as failed in the tx history.

Reply:
i see `useUpdateCctpTransactions` and `getUpdatedCctpTransfer`
seems like `useUpdateCctpTransactions` is not in use, so we should delete it

Reply:
removed